### DISCUSSION TITLE
有没有userscripts版本（油猴、暴力猴脚本版本）

### DISCUSSION THREAD

**Discussion by maoxuner** [PengChen96/ajax-tools#13] [Ideas]
如题。我想用油猴脚本来管理这些类型的插件，同时也方便在其他浏览器上使用。有没有响应的版本或者研发计划？

**Comment by PengChen96** [comment]
> 如题。我想用油猴脚本来管理这些类型的插件，同时也方便在其他浏览器上使用。有没有响应的版本或者研发计划？

目前没有这种油猴脚本版本，暂时没这计划，没怎么研究过油猴插件脚本，不知道能不能调用chrome的api。
